### PR TITLE
Add unique names for metrics collected. Bug fix - duplicate metronic instances.

### PR DESCRIPTION
**Pull request by arobson**
* Add metadata to metrics to provide a unique name for each metric collected
 * Bug fix - prevent default

### CODE REVIEW

File: src/http/adapter.js [hunk 1]
```diff
@@ -73,20 +73,18 @@ function getActionMetadata( state, resource, actionName, action, meta, resources
 	var alias = buildActionAlias( resource.name, actionName );
 	var resourceKey = [ [ resource.name, actionName ].join( '-' ), 'http' ];
 	var metricKey = [ state.metrics.prefix ].concat( resourceKey );
-	var errors = state.metrics.meter( resourceKey.concat( 'error' ) );
 	meta.routes[ actionName ] = { method: action.method, url: url };
 	return {
 		alias: alias,
 		authAttempted: function() {
-			state.metrics.authorizationAttempts.record();
+			state.metrics.authorizationAttempts.record( {1,  name: 'HTTP_AUTHORIZATION_ATTEMPTS' } );
```
Comment (brianedgerton) [inline src/http/adapter.js]: Syntax error here. The "1" needs to be outside that opening object brace.

**Comment by brianedgerton** [comment]
Tests passed after I fixed that syntax error on my local copy. Update that and I'll merge.

**Comment by arobson** [comment]
@brianedgerton :beers: and such. Thanks for catching that!

**Comment by brianedgerton** [comment]
That's what I do, yo. Tests are passing now. :fireworks:

**Pull request merged**